### ISSUE TITLE
cd lib/client && bower install

### ISSUE THREAD

**Issue by donlion** [rakuten-frontend/bower-browser#1]
Tried running "**sudo npm install bower-browser -g**" and recieved following:

```
npm ERR! bower-browser@0.2.0 postinstall: `cd lib/client && bower install`
npm ERR! Exit status 8
npm ERR! 
npm ERR! Failed at the bower-browser@0.2.0 postinstall script 'cd lib/client && bower install'.
npm ERR! This is most likely a problem with the bower-browser package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cd lib/client && bower install
npm ERR! You can get their info via:
npm ERR!     npm owner ls bower-browser
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
```

I can send you the log file on a mail, if you need it, but it's about 10.000 lines, so I thought not pasting it here, would ease the readability ;)

/Leo

**Comment by blackfyre** [comment]
Same here...
Additional info: OSX 10.10.1 Yosemite, using sudo to run npm from terminal.

<pre>
> bower-browser@0.2.0 postinstall /usr/local/lib/node_modules/bower-browser
> cd lib/client && bower install
Error: EACCES, permission denied '/Users/currentMachineUser/.config/configstore/insight-bower.yml'
You don't have access to this file.
    at Object.fs.openSync (evalmachine.<anonymous>:438:18)
    at Object.fs.readFileSync (evalmachine.<anonymous>:289:15)
    at Object.create.all.get (/usr/local/lib/node_modules/bower/node_modules/insight/node_modules/configstore/configstore.js:33:29)
    at Object.Configstore (/usr/local/lib/node_modules/bower/node_modules/insight/node_modules/configstore/configstore.js:26:44)
    at new Insight (/usr/local/lib/node_modules/bower/node_modules/insight/lib/index.js:26:34)
    at ensureInsight (/usr/local/lib/node_modules/bower/lib/util/analytics.js:26:19)
    at Object.setup (/usr/local/lib/node_modules/bower/lib/util/analytics.js:43:9)
    at Object.<anonymous> (/usr/local/lib/node_modules/bower/bin/bower:72:11)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
npm ERR! Darwin 14.0.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "-g" "bower-browser"
npm ERR! node v0.10.33
npm ERR! npm  v2.1.12
npm ERR! code ELIFECYCLE
npm ERR! bower-browser@0.2.0 postinstall: `cd lib/client && bower install`
npm ERR! Exit status 8
npm ERR! 
npm ERR! Failed at the bower-browser@0.2.0 postinstall script 'cd lib/client && bower install'.
npm ERR! This is most likely a problem with the bower-browser package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cd lib/client && bower install
npm ERR! You can get their info via:
npm ERR!     npm owner ls bower-browser
npm ERR! There is likely additional logging output above.
</pre>

**Comment by htanjo** [comment]
Thanks for reporting.
You may have a permission issue. It's better to install packages without `sudo`.

See: https://github.com/sindresorhus/guides/blob/master/npm-global-without-sudo.md

Currently bower-browser is designed for user command.
But I'm planning to support broad environment, or notify a message before the error.

**Comment by blackfyre** [comment]
This is definitely a permission error :smile:, tried it our on windows this morning and everything was working fine.

My guess would be that everybody will have this problem who didn't bother to change the npm global directory to a user accessible one on OSX and *nix systems.

They can solve this in a few ways:
- Set a new global directory for npm
- change the permissions for the current npm global and bin folders to allow sudo less access

**Comment by htanjo** [comment]
Hi! I've released `bower-browser v0.3.0` and this version supports `npm install` with `sudo`. :laughing:
